src/Views/Form/Themes/Sequoia/optionConfig.php

```diff
@@ -1,26 +1,40 @@
 <?php
+global $post;
 return [
 	'introduction'        => [
 		'name'   => __( 'Introduction', 'give' ),
 		'desc'   => __( 'Step description will show up here if any', 'give' ),
 		'fields' => [
+			[
+				'name'    => __( 'Include Introduction', 'give' ),
+				'desc'    => sprintf( __( 'Should this form include an introduction section?', 'give' ), '#' ),
```

Comment:
Why we are using `sprintf` here?

Reply:
Oops! Good catch, I have now removed this sprintf call. Ref: https://github.com/impress-org/givewp/commit/4444088ee4a560c9a0406e3bd1e91bafcf5d8172